TACC/TACC-Docs
# chore: example change to remove corral/corral url

## Status

Solved on Feb 2, 2023 (5b93b39) by @susanunit reconfiguring the makefiles.

## Overview

Proposal for one way to remove the duplicate system names in URLs e.g. [`/hpcugs/corral/corral/`](https://docs.tacc.utexas.edu/hpcugs/corral/corral/).

⚠️ This is just a proposal PR, to record one way to resolve this problem, with minimal live interaction with other people.

⚠️ This PR is not meant to be merged. It does not account for `corral/corralsrc.md` nor `corral/makefile`.

## Related

None

## Changes

- rename `corral.md` to `index.md`
- update one reference (there are more references)

## Testing

0. Be on `main` branch.
1. Locally, run `docker-compose up`.
1. Open https://docs.tacc.utexas.edu/hpcugs/corral/.
1. Verify page does **not** load.
1. Locally, run `docker-compose down`.
0. Checkout `task/example-change-to-remove-corral-corral-url` branch.
1. Locally, run `docker-compose up`.
1. Open https://localhost:8000/hpcugs/corral/.
1. Verify page **does** load.

## UI

Skipped.
